elements/lisk-api-client/src/resources/delegates.ts

```diff
@@ -31,15 +31,15 @@ export class DelegatesResource extends APIResource {
 
 		this.get = apiMethod({
 			defaultData: {
-				sort: 'rank:asc',
+				sort: 'username:asc',
 			},
 			method: GET,
 		}).bind(this);
 
 		this.getStandby = apiMethod({
 			defaultData: {
 				offset: 101,
-				sort: 'rank:asc',
+				sort: 'username:asc',
```

Comment:
AFAIK, here is a 🐛, @shuse2 

offset: 101, sorted by `username:asc` doesn't return standby delegates.

It could work with `sort: 'voteWeight:desc`